docker-compose.yml

```diff
@@ -40,10 +40,17 @@ services:
       - ./scripts/init-celestia-bridge.sh:/scripts/init.sh:ro
       - celestia-bridge:/home/celestia
     entrypoint: /scripts/init.sh
-    command: celestia bridge start --log.level.module share/discovery:fatal
+    command: celestia bridge start --log.level.module share/discovery:fatal --keyring.keyname node --rpc.skip-auth
     ports:
       - "26658:26658"
       - "2121:2121"
+    healthcheck:
```

Comment:
sneaking in this healthcheck and enabling `--rpc.skip-auth` as will be useful adding rollkit/reth